config/api_platform/resources.yaml

```
@@ -17,6 +17,69 @@ resources:
+                        type: string
+                        example: 'sess_1234567890abcdef'
+            '400':
+              description: 'Validation error'
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      errors:
+                        type: object
+                        example: { 'email': 'This field is required' }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)